submissions/franchukv/js-oop-frogger/app.js

```diff
@@ -0,0 +1,136 @@
+const CANVAS_HEIGHT = 606;
+const CANVAS_WIDTH = 505;
```

Comment:
Looks like there is correlation between tile width and canvas dimensions. Let computer do the calcs.